coalib/bearlib/abstractions/Linter.py

```diff
@@ -207,6 +207,9 @@ def check_prerequisites(cls):
             :return:
                 True if operational, otherwise a string containing more info.
             """
+            if not super(LinterBase, cls).check_prerequisites():
```

Comment:
@userzimmermann @SanketDG  Oh yeah, sorry that should be done :) . I'll add that and one test case :).